docstore/awsdynamodb/create_tables.sh

```diff
@@ -36,6 +37,7 @@ aws dynamodb create-table \
 # The docstore-test-2 table has both a partition and a sort key, and two indexes.
 
 aws dynamodb create-table \
+  --region us-east-2 \
```

Comment:
Why is this line different?

Reply:
The unit tests hard code the region as us-east-2, but the creation script used to set up the test db's don' t specify a region, falling back to the user default, this was to improve consistency in testing